redis.conf

```diff
@@ -1029,11 +1029,11 @@ acllog-max-len 128
 # ACL file, the server will refuse to start.
 #
 # The format of the external ACL user file is exactly the same as the
-# format that is used inside redis.conf to describe users.
+# format that is used inside valkey.conf to describe users.
 #
-# aclfile /etc/redis/users.acl
+# aclfile /etc/valkey/users.acl
 
-# IMPORTANT NOTE: starting with Redis 6 "requirepass" is just a compatibility
```

Comment:
Same as above, there is no Valkey 6 version, pls update this information properly.